conans/cli/cli.py

```diff
@@ -81,13 +81,15 @@ def __init__(self, conan_api):
         self._commands = {}
         conan_commands_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "commands")
         for module in pkgutil.iter_modules([conan_commands_path]):
-            self._add_command("conans.cli.commands.{}".format(module.name), module.name)
+            module_name = module[1]
```

Comment:
Is this a Py2.7 thing? If that is the case, I am happy with the check you did in the test to run the test only in Py3

Reply:
This is because in 3.5 they changed the return from a tuple to a named tuple for `iter_modules` return value:
https://docs.python.org/3.5/library/pkgutil.html#pkgutil.iter_modules
https://docs.python.org/3.6/library/pkgutil.html#pkgutil.iter_modules
Python2 issue is another one.